lib/improver/tests/wind_direction/test_WindDirection.py

```diff
@@ -97,13 +97,91 @@
                             [0.984807753, 0.984807753]])
 
 
+def make_wdir_cube_222():
+    """Make a wind direction cube for testing this plugin"""
+    # 2x2x2 3D Array containing wind direction in angles.
+    data = WIND_DIR_DEG
```

Comment:
Now that we're working in cubes rather than arrays, can we get rid of (at least some of) these global variables, or incorporate them into "setUp" functions?